src/database.c

```diff
@@ -365,8 +365,10 @@ log_database_get_previous_chat(const gchar* const contact_barejid, const char* s
         char* to_jid = (char*)sqlite3_column_text(stmt, 3);
         char* type = (char*)sqlite3_column_text(stmt, 4);
         char* encryption = (char*)sqlite3_column_text(stmt, 5);
+        char* id = (char*)sqlite3_column_text(stmt, 6);
 
         ProfMessage* msg = message_init();
+        msg->id = id ? strdup(id) : NULL;
```

Comment:
I see you introduced `STRDUP_OR_NULL` in `buffer.c`. I wonder if it would be worth it to have this in common.h and use it here (and in other places) as well? @sjaeckel ?

Reply:
@sjaeckel ping